src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi

```diff
@@ -71,6 +74,11 @@ cdef class _AioCall:
             c_deadline,
             NULL
         )
+        if credentials:
+            set_credentials_error = grpc_call_set_credentials(self._grpc_call_wrapper.call, credentials.c())
+            if set_credentials_error != GRPC_CALL_OK:
+                raise Exception("Credentials couldn't have been set")
```

Comment:
nit: Should we use `RuntimeError` instead of `Exception`.

Reply:
Ping. If possible, please include the error code in the exception message. It could make our life easier in future.